src/foam/nanos/pom.js

```diff
@@ -25,7 +25,8 @@ foam.POM({
     { name: "jetty/pom"},
     { name: "ruler/pom"},
     { name: "referral/pom" },
-    { name: "so/pom"}
+    { name: "so/pom"},
+    { name: "place/pom"},
```

Comment:
These are in alphabetic order, please move above ruler/pom.  Thanks.

Reply:
You mean sort the array right?